alivee/utils/requests/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class RequestMethodNotSupportedException(Exception):
+    pass
```

Comment:
Can we also record what is the current method (make it a param in `__init__`) so we know what method we are blocking and can have more info in the exception?